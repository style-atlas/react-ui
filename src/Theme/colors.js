import { red, lightBlue, blueGrey, orange } from "material-ui/colors";

export { red, lightBlue, blueGrey, orange };
