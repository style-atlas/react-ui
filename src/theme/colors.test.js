import React from "react";
import * as colors from "./colors";

test("Colors should be as snapshot defines", () => {
  expect(colors).toMatchSnapshot();
});
