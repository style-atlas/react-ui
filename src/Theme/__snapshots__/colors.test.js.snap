// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors should be as snapshot defines 1`] = `
Object {
  "blueGrey": Object {
    "100": "#cfd8dc",
    "200": "#b0bec5",
    "300": "#90a4ae",
    "400": "#78909c",
    "50": "#eceff1",
    "500": "#607d8b",
    "600": "#546e7a",
    "700": "#455a64",
    "800": "#37474f",
    "900": "#263238",
    "A100": "#cfd8dc",
    "A200": "#b0bec5",
    "A400": "#78909c",
    "A700": "#455a64",
  },
  "lightBlue": Object {
    "100": "#b3e5fc",
    "200": "#81d4fa",
    "300": "#4fc3f7",
    "400": "#29b6f6",
    "50": "#e1f5fe",
    "500": "#03a9f4",
    "600": "#039be5",
    "700": "#0288d1",
    "800": "#0277bd",
    "900": "#01579b",
    "A100": "#80d8ff",
    "A200": "#40c4ff",
    "A400": "#00b0ff",
    "A700": "#0091ea",
  },
  "orange": Object {
    "100": "#ffe0b2",
    "200": "#ffcc80",
    "300": "#ffb74d",
    "400": "#ffa726",
    "50": "#fff3e0",
    "500": "#ff9800",
    "600": "#fb8c00",
    "700": "#f57c00",
    "800": "#ef6c00",
    "900": "#e65100",
    "A100": "#ffd180",
    "A200": "#ffab40",
    "A400": "#ff9100",
    "A700": "#ff6d00",
  },
  "red": Object {
    "100": "#ffcdd2",
    "200": "#ef9a9a",
    "300": "#e57373",
    "400": "#ef5350",
    "50": "#ffebee",
    "500": "#f44336",
    "600": "#e53935",
    "700": "#d32f2f",
    "800": "#c62828",
    "900": "#b71c1c",
    "A100": "#ff8a80",
    "A200": "#ff5252",
    "A400": "#ff1744",
    "A700": "#d50000",
  },
}
`;
